### DISCUSSION TITLE
好可愛的龍

### DISCUSSION THREAD

**Discussion by Andrea0126** [share121/inter-knot#5466] [原神]
![IMG_20240928_212707](https://github.com/user-attachments/assets/cb35bdc4-b048-48d1-9c8d-d010856ef19c)
![IMG_20240928_212652](https://github.com/user-attachments/assets/29b208ed-badf-4d91-bd2e-e4ae7d241385)
在雪山和下雨天都有不同的待機動作，好可愛😍

**Comment by Allium0** [comment]
这个是怎么获得的

**Reply by Andrea0126** [reply]
這是官方送原石的時候一起給的，去收信就可以了
![Screenshot_2024-09-29-10-15-07-751_com miHoYo GenshinImpact](https://github.com/user-attachments/assets/3b5b990f-1798-4bb1-b3e5-ba7591275aca)
